types/indy-sdk/tsconfig.json

```diff
@@ -2,7 +2,8 @@
     "compilerOptions": {
         "module": "commonjs",
         "lib": [
-            "es6"
+            "es6",
+            "ESNext.BigInt"
```

Comment:
Sorry, I forgot to ask about this earlier too: is it needed?

Reply:
Yes. the `buffer` library relies on it

Reply:
That's not good, since it's basically requires your consumers to use `BigInt` too.  It seems to be a problem in `buffer`, so I made [this PR](https://github.com/feross/buffer/pull/299), but no idea how live is that project so making it possible to merge this in the meantime.

Reply:
I'll merge for now, but will make an additional PR once your PR is merged.

Thank you!